packages/commonwealth/server/migrations/20240610170024-add-osmosis-chainnode.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
What is the purpose of this migration - will it overwrite existing prod configs for osmosis?

Reply:
Same as Evmos - this tries to add the production Osmosis ChainNode if it's missing, eg running from a fresh db:create.

Otherwise Cosmos logins using Osmosis APIs don't work and we encounter many weird edge cases around wallet login being broken.